INSTALL.txt

```diff
@@ -8,60 +8,26 @@ source activate $DEMO_ENV_NAME
 ipython kernel install --name $DEMO_ENV_NAME --display-name $DEMO_ENV_NAME --sys-prefix
 
 # Install most recent notebook
-conda install -y -c conda-forge notebook=4.4.1
+conda install -y -c conda-forge notebook=5.0.0
 
-# Install jupyterlab
-conda install -y -c conda-forge jupyterlab=0.16.2
-
-# Install ipywidgets
-conda install -y -c conda-forge ipywidgets=6.0.0
+# Install node
+conda install -c conda-forge nodejs=6.11.0
 
-# Install the jupyterlab widgets plugin
-pip install jupyterlab_widgets==0.6.15
-jupyter labextension install --sys-prefix --py jupyterlab_widgets
-jupyter labextension enable jupyterlab_widgets --py --sys-prefix
+# Install jupyterlab
+conda install -y -c conda-forge jupyterlab=0.24.1
 
-# GeoJSON map viewer
```

Comment:
Added those back in.  Also switched graphviz to conda since Travis isn't using homebrew. @jasongrout mentioned that graphviz w/ conda might not work but I didn't see any problems?